pkg/build/controller/strategy/custom_test.go

```diff
@@ -116,6 +116,7 @@ func mockCustomBuild() *buildapi.Build {
 						{Name: "FOO", Value: "BAR"},
 					},
 					ExposeDockerSocket: true,
+					ForcePull: true,
```

Comment:
I thought it best to be consistent in explicitly setting ForcePull like the test explicitly set ExposeDockerSocket, and did not see how the `TestCustomCreateBuildPod` was leveraging ExposeDockerSocket in an explicit  way, and hence for that to mean a benign setting was viewed as a acceptable test.

Now if ExposeDockerSocket needs to  be true for `strategy.GetBuildPod(expected) to work, and hence it *needs* to be explicitly set, shouldn't its default be set to true in types.go?  Or perhaps I'm missing something wrt bool defaults and/or how the custom build strategy works.

With that explanation, I'm certainly fine with removing it.  I'll admit to some amount of curiosity as to why ExposeDockerSocket was set to true, and that it might get flushed out when reviewing the PR.